<aura:component implements="force:appHostable,flexipage:availableForAllPageTypes,flexipage:availableForRecordHome,force:hasRecordId,forceCommunity:availableForAllPageTypes,force:lightningQuickAction" access="global" >
	
    <aura:attribute name="accountRecord" type="Object"/>
      <aura:attribute name="showerror" type="string" default ="nothing"/>
     <aura:attribute name="recordLoadError" type="String"/>
        <aura:attribute name="getcallbackcheck" type="String" default ="cool"/>
    <force:recordData aura:id="recordLoader"
    recordId="0015j00001dbbOtAAI"
    fields="Name,Description,Phone,Industry"
    targetFields="{!v.accountRecord}"
                      targetError ="{!v.showerror}"
 mode="EDIT"
                   
    />
    FORCE RECORD DATATATATATATATATTATTATATATTA
     what is the error  {!v.showerror}
    <div>
        <lightning:card iconName="standard:account" title="{!v.accountRecord.Name}" >
            <div class="slds-p-horizontal--small">
                <p class="slds-text-heading--medium"><lightning:input title="Phone" value="{!v.accountRecord.Phone}" /></p>
                <p class="slds-truncate"><lightning:input title="Description" value="{!v.accountRecord.Description}" /></p>
                <p class="slds-truncate"> <lightning:input title="Industry" value="{!v.accountRecord.Industry}" /></p>
                <!--  <p class="slds-truncate"> <lightning:input title="Rating" value="{!v.accountRecord.Rating}" /></p>  -->
            </div>
        </lightning:card>
    </div>
    <lightning:button label="Save" onclick="{!c.handlesave}"/>
     {!v.getcallbackcheck}
</aura:component>